README.md

```diff
@@ -21,14 +21,27 @@ npm install @zendeskgarden/svg-icons
 The package contains a `src` folder with individual SVG icons along with
 a combined "spritemap" `dist` of SVG symbols.
 
+## Bundling & Configuration
+
+This library delivers icons as raw SVG files. To consume them directly as React components or import them in your JavaScript, your build environment must be configured to transform SVG assets.
+
+While many modern frameworks handle this automatically, custom setups may require specific plugins.
```

Comment:
```suggestion
This library delivers icons as raw SVG files. To consume them directly as React components or import them in your JavaScript, your build environment must be configured to transform SVG assets. While many modern frameworks handle this automatically, custom setups may require specific plugins.
```

[nit] this can be a single paragraph